welcome/index.adoc

```diff
@@ -206,9 +206,7 @@ xref:../operators/operator_sdk/helm/osdk-helm-support.adoc#osdk-helm-support[Hel
 - **Reference the xref:../rest_api/index.adoc#api-index[REST API index]**: Learn about {product-title} application programming interface endpoints.
 
 // Need to provide a link closer to 4.15 GA
-- **Software Supply Chain Security enhancements**: The PipelineRun *details* page in the *Developer* or *Administrator* perspective of the web console provides a visual representation of identified vulnerabilities, which are categorized by severity. Additionally, these enhancements provide an option to download or view Software Bill of Materials (SBOMs) for enhanced transparency and control within your supply chain. Learn about setting up OpenShift Pipelines in the web console to view Software Supply Chain Security elements. 
-
-//When doc available, add a link to /openshift-pipelines/latest/secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements[Setting up OpenShift Pipelines in the web console to view Software Supply Chain Security elements]
+- **Software Supply Chain Security enhancements**: The PipelineRun *details* page in the *Developer* or *Administrator* perspective of the web console provides a visual representation of identified vulnerabilities, which are categorized by severity. Additionally, these enhancements provide an option to download or view Software Bill of Materials (SBOMs) for enhanced transparency and control within your supply chain. Learn about setting up OpenShift Pipelines in the web console to view Software Supply Chain Security elements. link:https://docs.openshift.com/pipelines/1.13/secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.html[Setting up OpenShift Pipelines in the web console to view Software Supply Chain Security elements]
```

Comment:
Just a question - is there a reason not to embed this link within the previous sentence?
```suggestion
- **Software Supply Chain Security enhancements**: The PipelineRun *details* page in the *Developer* or *Administrator* perspective of the web console provides a visual representation of identified vulnerabilities, which are categorized by severity. Additionally, these enhancements provide an option to download or view Software Bill of Materials (SBOMs) for enhanced transparency and control within your supply chain. Learn about link:https://docs.openshift.com/pipelines/1.13/secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.html[Setting up OpenShift Pipelines in the web console to view Software Supply Chain Security elements].
```